ui/apps/ui/src/app/components/search-input/search-input.component.ts

```diff
@@ -271,9 +271,10 @@ export class SearchInputComponent implements OnInit {
             .pipe(untilDestroyed(this))
         )
       )
-      .subscribe(
-        (suggestedResults) => (this.suggestedResults = suggestedResults)
-      );
+      .subscribe((suggestedResults) => {
+        this.suggestedResults = suggestedResults;
+        console.log(this.suggestedResults);
```

Comment:
Is this console log required? Or an artifact from debugging?

Reply:
I was looking for something for Maria and forgot about it